python/cudf/cudf/_lib/__init__.py

```diff
@@ -0,0 +1,29 @@
+from . import (
```

Comment:
We've avoided relative imports in general thus far but I'm guessing we need it here?

Reply:
Yes - I gave it a lot of thought and it's the cleanest way to make all the modules available as attributes of the `_lib` package.